src/components/labels/_labels.scss

```diff
@@ -108,6 +101,13 @@ $includeHtml: false !default;
       }
 
     }
+
+    &--unstyled {
```

Comment:
i just put it a lit lower, because it worked only with sg-label--small and didn't work with sg-label--secondary